db/seeds.rb

```diff
@@ -5,3 +5,14 @@
 #
 #   movies = Movie.create([{ name: 'Star Wars' }, { name: 'Lord of the Rings' }])
 #   Character.create(name: 'Luke', movie: movies.first)
+AmusementPark.create!(
```

Comment:
good! Not necessary for now, but doing a `find_by` with the name will allow for the seeds to be rerunnable. If the AmusementPark is not found, create it, otherwise skip it.